packages/encryption/src/messageSignature.ts

```diff
@@ -1,13 +1,13 @@
-import { sha256 } from 'sha.js';
-import { encode, decode, encodingLength } from 'varuint-bitcoin';
+import { sha256 } from '@noble/hashes/sha256';
 import { Buffer } from '@stacks/common';
+import { decode, encode, encodingLength } from 'varuint-bitcoin';
```

Comment:
Note: `varuint-bitcoin` is a good candidate for vendoring later, it has an unfriendly Nodejs buffer polyfill dependency.